out/main.go

```diff
@@ -60,7 +65,7 @@ func main() {
 		Number: v.String(),
 	}
 
-	err = bucket.Put(request.Source.Key, []byte(outVersion.Number), "text/plain", "")
+	err = bucket.Put(request.Source.Key, []byte(outVersion.Number), "text/plain", s3.BucketOwnerFull)
```

Comment:
Was this just to make the default consistent across providers? S3 defaults to 'private', but I don't think we cared too much about it.